site/_docs/continuous-integration.md

```diff
@@ -118,6 +118,7 @@ customizable. If your script won't change much, you can write your test
 incantation here directly:
 
 {% highlight yaml %}
+before_install: gem install jekyll html-proofer
```

Comment:
Gemfile and bundle install are good and that should be the mainline documentation. But for my use case "If your script won’t change much, you can write your test incantation here directly," and I don't want a Gemfile then this is the ticket!